pi_local.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This script will log in the address of a raspberry-pi 400 ubuntu server, copy the files over the Pi, excute the script, and SSH into the server.
+PI_HOST=pi.attlocal.net
+echo -e "\n==== Validate Pi server is running ====\n"
+while true
+do
+  if (  ping -c 1 $PI_HOST > /dev/null 2>&1 )
```

Comment:
Did you try the ssh w/ exit here?

Reply:
I did right after the our call but it wasnt working.